pkg/manager/impl/util/shared.go

```diff
@@ -291,5 +291,10 @@ func MergeIntoExecConfig(workflowExecConfig admin.WorkflowExecutionConfig, spec
 	if workflowExecConfig.GetInterruptible() == nil && spec.GetInterruptible() != nil {
 		workflowExecConfig.Interruptible = spec.GetInterruptible()
 	}
+
+	if !workflowExecConfig.GetOverwriteCache() && spec.GetOverwriteCache() {
```

Comment:
Interesting - so we can essentially disable caching for an entire project+domain?

Reply:
I thought about that, similar to how I previously added the `interruptible` flag, however I didn't see the exact use case of overwriting it to `false` (being able to distinguish between not set/`false`).
Since the main intended use case was for a single execution, I didn't anticipate we'd need the "tri-state" bool, especially since it's also less confusing in the console UI for end users.

@katrogan If you'd prefer to have the option, I can certainly adapt it since it'll be a minor change anyways.

Reply:
Yeah I suppose the project-domain and even application setting use cases are themselves probably very nice to haves that I can't imagine are used frequently. 👍 We can always revisit this if it ends up being a feature request